hypha/apply/projects/views/project.py

```diff
@@ -725,6 +725,7 @@ def get_context_data(self, **kwargs):
         return super().get_context_data(
             title=self.object.title,
             buttons=self.buttons(),
+            approval_form_exists=True if self.approval_form else False,
```

Comment:
This looks unnecessary here; it can be easily moved to template logic if `self.approval_form` is passed as context.

Reply:
As we don't need the approval_form in the template for any other usage so I thought it is better to pass just a boolean instead of the whole form? Maybe we can pass it later depending on the requirement.